packages/pds-ember/app/styles/pds/components/link/default/__config.scss

```diff
@@ -0,0 +1,51 @@
+$_module: "PDS.Components.Link.Default";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "sass:map";
+@use "../__config" as Link;
+@use "../../../theme";
+@use "../../../utils/pseudo" as Pseudo;
+
+// :any-link
+$DEFAULT: (
+  color: theme.$action-base,
+);
+// :any-link:hover
+$HOVER: map.merge($DEFAULT, (
+  borderBottomColor: currentcolor,
+));
+// :any-link:focus
+$FOCUS: map.merge($HOVER, (
+  boxShadow: theme.$boxShadow--focus,
+));
+// :any-link:active
+$ACTIVE: map.merge($HOVER, (
+  boxShadow: none !important,
+  color: theme.$action-d1,
+));
+
+:root {
+  @include Link.theme($DEFAULT);
+}
```

Comment:
NOTE: This will need refactored in a future work stream, in order to support container-based theming.